packages/slate-editor/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import React, { useState, type ReactNode } from 'react';
+
+import { Caret } from '#icons';
+
+import styles from './Accordion.module.scss';
+
+interface Props {
+    title: string;
+    children: ReactNode;
+}
+
+export function Accordion({ title, children }: Props) {
+    const [isOpen, setIsOpen] = useState(false);
+
+    function toggleOpen() {
+        setIsOpen(!isOpen);
+    }
+
+    return (
+        <div className={styles.Accordion}>
+            <button className={styles.button} onClick={toggleOpen}>
+                <span>{title}</span>
+                {
+                    <Caret
+                        className={classNames(styles.icon, {
+                            [styles.open]: isOpen,
+                        })}
+                    />
+                }
```

Comment:
Are `{ }` braces needed?

```suggestion
                <Caret
                    className={classNames(styles.icon, {
                        [styles.open]: isOpen,
                    })}
                />
```

Reply:
Updated